src/components/routes.js

```diff
@@ -4,18 +4,20 @@ import LoginContainer from "../containers/Login/LoginContainer";
 import SocialAuthContainer from "../containers/SocialAuthContainer";
 import Home from "./home";
 import RegisterContainer from "../containers/registerContainer";
+import { ProfilePage } from "../components/userProfile/ProfilePage";
+import EditProfileContainer from "../containers/profile/EditProfileContainer";
 
 class Routes extends Component {
   render() {
     return (
       <BrowserRouter>
-        <Switch>
-          <Route path="/" component={Home} exact={true} />
-          <Route path="/register" component={RegisterContainer} />
-          <Route exact path="/home" component={Home} />
-          <Route path="/login" component={LoginContainer} />
-          <Route path="/socialAuth" component={SocialAuthContainer} />
-        </Switch>
+        <Route path="/" component={Home} exact={true} />
```

Comment:
Great work @e-ian 
I see you removed the <Switch>
It is actually important so as to get the first matching url as opposed to all urls that may match the same link.
For example  /profile/edit would match /profile and /profile/edit without the <Switch>
With <Switch>  it only matches /profile/edit

Reply:
Thanks @patrickf949 . I have worked on this issue. It is now fine.